ui/src/transaction.ts

```diff
@@ -481,6 +611,19 @@ export const formatDate = (timestamp: number, option: string): string => {
   return `${time.toLocaleString(undefined, options)}`;
 };
 
+export const nFormatter = (num: number): string => {
```

Comment:
We name all of the other formatting functions as verbs: `formatPayer`, `formatSubject`, I think this would make more sense as `formatNumber`

Reply:
fixed, renamed it to `formatRad` :+1: